teps/0128-scheduled-runs.md

```diff
@@ -199,8 +200,73 @@ to address specifically.
 feature like [polling runs](./0083-polling-runs-in-tekton.md). This proposal doesn't prevent that,
 and ScheduledTriggers could be leveraged to implement similar functionality in a higher level system.
 
+- This seems to templatisation over templatisation. Pipelinerun already give us flexibility to pass parameters for Pipeline based on requirements. TriggerBindings and TriggerTemplates were designed for templatising Pipelineruns based on events coming. In this case, they will always remain same.
+
 ## Alternatives
 
+### Add a field or annotation in Pipelinerun for scheduling
+
+In this case we add a controller in experimental repo which can later be added to pipelines. Or it can be part of Pipelinerun controller.
+We will add field for scheduling:
+```yaml
+apiVersion: tekton.dev/v1
+kind: Pipelinerun
+metadata:
+  generateName: nightly-release
+spec:
+  schedule: "0 0 * * *"
```

Comment:
This proposal breaks the fundamental assumption that one `xRun` object is associated with one execution of that resource. The only slight exception that we have to that model today is "retries" - which in any case are still part of the same single execution.

If we had a long-running `PipelineRun` associated with multiple executions, we would have to significantly change the affinity assistant, the way we manage PVC, and for `TaskRuns` the way we manage pods, logs etc. 
It would be a breaking change for all tools like CLI and dashboard too.

I think a better approach is a dedicated CRD for that. Whether that is should be managed by the triggers controller, the pipeline controller or a brand-new controller I'm not sure thought :D

Reply:
That's why I added a second proposal about `ScheduledTrigger`  or `ScheduledPipelinerun` also.